analyzer/src/main/kotlin/managers/Conan.kt

```diff
@@ -144,25 +142,47 @@ class Conan(
         stashDirectories(directoryToStash).use {
             configureRemoteAuthentication(conanConfig)
 
+            // TODO: Support lockfiles which are located in a different directory than the definition file.
+            val lockfileName = analyzerConfig.packageManagers?.get(managerName)?.options?.get("lockfileName")
```

Comment:
Please document the `lockfileName` configuration option.
You can do this in the reference.yml in the package manager section:
https://github.com/oss-review-toolkit/ort/blob/509aeb2b3d21a2778e668e018401b448192c3e3f/model/src/main/resources/reference.yml#L79

Reply:
The option also needs to be documented in the docs of the Conan class, for an example see `Pip.kt`.